src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -237,30 +240,27 @@ public Path getJarPath(String jarName) {
    */
   public void validate() throws AppEngineException {
     if (sdkPath == null) {
-      throw new AppEngineException("Validation Error: SDK path is null");
```

Comment:
We're using CloudSdkResolvers to get the SDK path. It could be possible that some of them incorrectly return null, so we should keep the null check on validate(). Unless you want to add a @NotNull annotation to CloudSdkResolver.getCloudSdkPath(), which I'm not sure is advisable or not.